MODULE.bazel

```
@@ -1,5 +1,5 @@
 module(
-    name = "codeql",
+    name = "ql",
     version = "0.0",
 )
```

I think we can avoid having to change `@codeql` to `@ql` by using `repo_name` here as well:
```suggestions
module(
    name = "ql",
    version = "0.0",
    repo_name = "codeql",
)

---

This is an excellent and unexpected find, thanks! This indeed makes things much nicer.